pywbem/cim_operations.py

```diff
@@ -861,6 +890,100 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_server_response_time = None
 
     @property
+    def operation_recorder(self):
+        """
+        This is a writeable property; setting this property will cause the
+        defined recorder to be activated.
+
+        Effective pywbem 0.10.0 operation_recorder became a property rather
+        than an attribute. The getter operation returns TODO
+        The setting operation executes the add_operation_recorder with the
+        value provided. Since there can be multiple simultaneous recorders,
+        setting operation_recorder to None is no longer valid.
+
+
+        **Experimental:** This property is experimental for this release.
+
+        **Deprecated:** This property has been marked deprecated.  The method
+        add_operation_recorder should be used to activate an operation
+        recorder.
+        """
+        warnings.warn(
+            "Directly setting operation recorder has been deprecated",
+            DeprecationWarning)
+        return self._operation_recorder
+
+    @operation_recorder.setter
+    def operation_recorder(self, value):
```

Comment:
The docstring of this property setter is not generated in the documentation, so we might as well remove the docstring and replace it with a comment such as "# For documentation, see property operation_recorder."

Reply:
Left the docstring but changed the text and moved the Parameters, exceptions documentation to the property.